knexfile.js

```diff
@@ -17,6 +17,9 @@ const pgConfig = {
   migrations: {
     tableName: 'migrations',
   },
+  seeds: {
+    directory: './seeds/',
+  },
```

Comment:
this shouldn't be needed? Does it error without?

Reply:
no, it can be removed and it has been, at first added into test/ folder but then looked at the migrations so followed default so this is no longer relevant as covered in the knex's docs